branca/plugins/__init__.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
We don't need these? I thought they were necessary when any special characters are used in the file.